web-app/src/make_celery.py

```diff
@@ -0,0 +1,4 @@
+from app import create_app
+
+flask_app = create_app()
+celery_app = flask_app.extensions["celery"]
```

Comment:
What uses `make_celery.py`? Is it loaded by convention?

Reply:
how to start celery: celery -A make_celery worker --loglevel INFO